frontend/test/metabase/scenarios/native/native_subquery.cy.spec.js

```diff
@@ -263,4 +267,33 @@ describe("scenarios > question > native subquery", () => {
       cy.contains("41");
     });
   });
+
+  it.skip("should be able to reference a nested question (metabase#25988)", () => {
+    const questionDetails = {
+      name: "Nested GUI question",
+      query: {
+        "source-table": "card__1",
+        limit: 2,
+      },
+    };
+
+    cy.createQuestion(questionDetails).then(
+      ({ body: { id: nestedQuestionId } }) => {
+        const tagID = `#${nestedQuestionId}`;
+
+        startNewNativeQuestion();
+        SQLFilter.enterParameterizedQuery(`SELECT * FROM {{${tagID}`);
+
+        cy.findByTestId("sidebar-header-title")
+          .invoke("text")
+          .should("eq", questionDetails.name);
+
+        runNativeQuery({
```

Comment:
Could we rely on some results present, and not on the backend not error-ing?

Reply:
Absolutely.
That's why line 295 is there for.

Reply:
I still don't think we should assert for a dataset response here

Reply:
I'm happy to remove that assertion but am genuinely interested to learn why do you think that?
Are there any downsides to this approach that I'm not seeing?

Reply:
Removed, btw
https://github.com/metabase/metabase/pull/28135/commits/984775a4b5bceb1cbfa1d22509a6466ebac93535